README.md

```diff
@@ -262,30 +262,127 @@ Parser consists of the following phases:
 ## Used technologies
 
 ### Extensions / Plugins
+main plugins:
+
+* React - https://reactjs.org/
```

Comment:
maybe simply:

```
* [React](https://reactjs.org/)
* [Redux](https://redux.js.org/)
* [Webext](https://github.com/tshaddix/webext-redux)
* [styled-components](https://styled-components.com/)
* [vis-network](https://visjs.github.io/vis-network/docs/network/)
* [vis-timeline](https://visjs.github.io/vis-timeline/docs/timeline/)
* [unique-selector](https://github.com/ericclemmons/unique-selector)
* [renderjson](https://github.com/caldwell/renderjson)
* [react-fontawesome](https://github.com/FortAwesome/react-fontawesome)
* [Jest](https://jestjs.io/docs/en/cli.html)
* [Enzyme](https://enzymejs.github.io/enzyme/)
* [semver](https://github.com/npm/node-semver#readme)
* [Babel](https://babeljs.io/)
* [Eslint](https://eslint.org/)
* [Webpack](https://webpack.js.org/)
```

Can we divide them in categories:
- application / components
- testing / quality / code conventions
- delivery / deployment